### PR TITLE
[DFFE-156] Build a To-Do App with REACT

### PR DESCRIPTION
**Pull request by lorlahdey**
@Ireti203 

Web View
Capture1
Mobile View
Capture2

### CODE REVIEW

**Comment by Ireti203** [comment]
This PR  does not have the expected content.

**Comment by lorlahdey** [comment]
@Ireti203 I don't know why the PR is not showing the content, it's all in my submission folder under my name

**Comment by lorlahdey** [comment]
@Ireti203 the PR now contains expected contents

File: submissions/lawalLolade/React/to-do-app/src/App.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+.todo-header {
+  background-color: #353c3c;
+  height:300px;
+  color: #fff;
+  padding: 10px;
+}
+.todo-header-content{
+  width: 55%;
+  margin: 60px auto;
+  /* padding-bottom: 10px; */
```
Comment (Ireti203) [inline submissions/lawalLolade/React/to-do-app/src/App.css:16]: Remove commented out code

File: submissions/lawalLolade/React/to-do-app/src/components/form.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+        const newItem ={
+            text:this.state.inputText,
+            completed: false,
+        }
+
+        this.props.onSubmit(newItem)   
+
+        this.setState( (prevState)=>({
+            // todos:prevState.todos.concat(newItem),
```
Comment (Ireti203) [inline submissions/lawalLolade/React/to-do-app/src/components/form.js:29]: remove commented out code

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**